### PR TITLE
feat(tickets): added child and student ticket types

### PR DESCRIPTION
**Pull request by paperclypse**
## What/Why/How?
It seemed like it would be a good idea to add museum ticket types for students (reduced price) and children aged 0-5 (free) so that there are a variety of ticket prices.

(Created as part of the Write The Docs writing conference Writing Day)

## Check yourself

- [ x] Code is linted
- [ ] Tested with redoc/reference-docs/workflows (internal)
- [ ] All new/updated code is covered with tests

## Security

- [ ] Security impact of change has been considered
- [ ] Code follows company security practices and guidelines

### CODE REVIEW

File: openapi.yaml [hunk 1]
```diff
@@ -70,7 +70,7 @@ paths:
           $ref: '#/components/responses/NotFound'
     get:
       summary: List special events
-      description: Return a list of upcoming special events at the museum.
+      description: Return a list of past special events at the museum. See one you like? Use this API to [buy a ticket](/#tag/Tickets/operation/buyMuseumTickets).
```
Comment (lornajane) [inline openapi.yaml:73]: I think this should still be upcoming special events rather than past ones - we may need to add a date filter and/or some further-in-the-future events!

And the URL should be a full URL, so that the description markdown is useful in any context.

File: openapi.yaml [hunk 2]
```diff
@@ -221,11 +229,13 @@ paths:
 components:
   schemas:
     TicketType:
-      description: Type of ticket being purchased. Use `general` for regular museum entry and `event` for tickets to special events.
+      description: Type of ticket being purchased. Use `general` for regular museum entry and `event` for tickets to special events. Use `student` for school-aged children. Visitors aged 0-5 are free.
```
Comment (lornajane) [inline openapi.yaml:232]: For clarity, could we rephrase this to put the general, student and child tickets in one sentence, and the special event ones in another? I think it's age-related general entry, or an event ticket.

**Review by lornajane — COMMENTED** [review]
More examples are always valuable, thank you! I added some comments for minor alterations.

File: openapi.yaml [hunk 3]
```diff
@@ -221,11 +229,13 @@ paths:
...
       type: string
       enum:
         - event
         - general
+        - student
+        - child
```
Comment (adamaltman) [inline openapi.yaml:238]: What if a child or student goes to an event?
